tests/commands/test_commit_command.py

```diff
@@ -52,6 +52,26 @@ def test_commit_retry_fails_no_backup(config, mocker):
     assert NoCommitBackupError.message in str(excinfo.value)
 
 
+@pytest.mark.usefixtures("staging_is_clean")
+def test_commit_allow_empty(config, mocker):
+    prompt_mock = mocker.patch("questionary.prompt")
+    prompt_mock.return_value = {
+        "prefix": "feat",
+        "subject": "user created",
+        "scope": "",
+        "is_breaking_change": False,
+        "body": "closes #21",
+        "footer": "",
+    }
+
+    commit_mock = mocker.patch("commitizen.git.commit")
```

Comment:
@Dranaxel Thanks for correcting me. Yes, you're right. In that case, we probably should assert whether this function is called with `--allow-empty`. This can be done by something like`assert_called_with`.

Reply:
Sorry i'm not sure about what you mean/how to use assert_called_with. Are you suggesting to assert the flag allow-empty was here when le command was launched ?

Reply:
assert `--allow-empty` is actually passed into `commitizen.git.commit` when the operation above is executed